Samples/CheckItem-By-Property/MainWindow.xaml

```diff
@@ -25,11 +25,11 @@
                                  Margin="20">
             
                         <!--Binding the IsChecked property from ViewModel-->
-                        <syncfusion:CheckListBox.ItemContainerStyle>
+                        <syncfusion:CheckListBox.Resources>
```

Comment:
Please remove unwanted code in the app config.
Also, get this issue fixed in all other checkbox samples.

Reply:
Removed unwanted code in the app config and this issue is not present in all other CheckedListBox sample.